libs/vkd3d/command.c

```diff
@@ -5440,7 +5454,8 @@ static void STDMETHODCALLTYPE d3d12_command_list_IASetIndexBuffer(d3d12_command_
 
     if (!view)
     {
-        WARN("Ignoring NULL index buffer view.\n");
+        WARN("Got NULL index buffer view, indexed draw calls will be dropped.\n");
```

Comment:
Does this actually unbind the current index buffer or should we just ignore the call entirely?

Reply:
Can test

Reply:
It does unbind the index buffer, yes.